snap/README.md

```diff
@@ -0,0 +1,30 @@
+# EdgeX REST Device Service Snap
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-device-rest)
+
+This folder contains snap packaging for the EdgeX REST Protocol Device Service Snap
+
+The snap currently supports both `amd64` and `arm64` platforms.
+
+
+## Snap configuration
+
+Device services implement a service dependency check on startup which ensures that all of the runtime dependencies of a particular service are met before the service transitions to active state.
+
+The logic implements a basic retry loop with a maximum duration and sleep interval. If the dependent services are not available, the service sleeps for the defined interval (default: 1s) and then tries again up to a maximum duration (default: 60s). These values can be overridden with the following commands:
+    
+To change the maximum duration, use the following command:
+
+```bash
+$ sudo snap set edgex-device-rest startup-duration=60
+```
+
+To change the interval between retries, use the following command:
+
+```bash
+$ sudo snap set edgex-device-rest startup-interval=1
+```
+
+To apply the settings, the service should then be restarted as follows:
+```bash
+$ sudo snap restart edgex-device-rest.device-rest-go
+```
```

Comment:
In your PR for device-camera, you provide instructions for starting the service, whereas here you're saying you need to restart to pickup the changes. I think in reality you probably need to combine the two (i.e. mention that device services are typically disabled on install as they require configuration to be provided first). Also note if/when we add rich configuration hooks to device services we can have the install hook check to see if any default configuration has been provided, and if so, start the service.

Reply:
OK, so time for me to check my glasses. I mistakenly assumed that device-rest was like the other device service snaps and installs with the service disabled. Turns out that's not the case, so please disregard my previous comment.